sdks/java/io/rabbitmq/src/test/java/org/apache/beam/sdk/io/rabbitmq/RabbitMqIOTest.java

```diff
@@ -98,11 +121,11 @@ public void testReadQueue() throws Exception {
             MapElements.into(TypeDescriptors.strings())
                 .via(
                     (RabbitMqMessage message) ->
-                        new String(message.getBody(), StandardCharsets.UTF_8)));
```

Comment:
There was previously much copy-pastad conversions between String and byte[]. I've moved these to the new Utils class and re-used them wherever I could.